charts/cloudzero-agent/README.md

```diff
@@ -36,30 +41,82 @@ helm install <RELEASE_NAME> cloudzero/cloudzero-agent \
     --set region=<REGION>
 ```
 
-### Secret Management
+### Mandatory Values
+
+There are several mandatory values that must be specified for the chart to install properly. Below are the required settings along with strategies for providing custom values during installation:
+
+| Key               | Type   | Default               | Description                                                                                                             |
+|-------------------|--------|-----------------------|-------------------------------------------------------------------------------------------------------------------------|
+| cloudAccountId    | string | `nil`                 | Account ID in AWS or Subscription ID in Azure where the cluster is running. Must be a string due to Helm limitations.  |
+| clusterName       | string | `nil`                 | Name of the cluster. Must be RFC 1123 compliant.                                                                         |
+| host              | string | `"api.cloudzero.com"` | CloudZero host to send metrics to.                                                                                      |
+| apiKey            | string | `nil`                 | The CloudZero API key to use for exporting metrics. Only used if `existingSecretName` is not set.                       |
+| existingSecretName| string | `nil`                 | Name of the secret that contains the CloudZero API key. Required if not providing the API key via `apiKey`.             |
+| region            | string | `nil`                 | Region where the cluster is running (e.g., `us-east-1`, `eastus`). For more information, see AWS or Azure documentation. |
+
+#### Overriding Default Values
+
+Default values are specified in the chart's `values.yaml` file. If you need to change any of these values, it is recommended to create a `values-override.yaml` file for your customizations.
+
+##### Using the `--values` Flag
+
+You can use the `--values` flag in your Helm commands to override values in the chart with a new file. Specify the name of the file after the `--values` flag:
```

Comment:
nit - this blurb uses the long form version of the option (`--values`), but the example uses the short version (`-f`). might be nice to have them be the same version for readers who may not be so familiar with helm install options